### PR TITLE
fix(window.name): Change format of window.name data to reflect that o…

### PR DESCRIPTION
**Pull request by arw180**
…f owf

### CODE REVIEW

**Comment by clarkjefcoat** [comment]
@arw180 - Is this fix going to be missing a closing "}" ?

File: src/java/marketplace/hal/AbstractWindownameWriter.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
     private static final byte[] windownamePrelude = (
         "<!DOCTYPE html>" +
         "<title>Windowname transport</title>" +
-        "<script>window.name='"
+        "<script>window.name='{status: 200, data: "
     ).getBytes(CHARSET);
 
     private static final byte[] windownamePostlude = ("';</script>").getBytes(CHARSET);
```
Comment (Kevin-K) [inline src/java/marketplace/hal/AbstractWindownameWriter.java]: add }:

`private static final byte[] windownamePostlude = ("}';</script>").getBytes(CHARSET);`

**Comment by arw180** [comment]
Added the missing `}`

**Comment by Kevin-K** [comment]
:+1:

**Comment by clarkjefcoat** [comment]
I can confirm that it compiles and the change looks reasonable, but don't have a legacy widget to test with.  :+1:?

**Comment by Kevin-K** [comment]
:+1: DELETE/GET/SET all look good for preferences

**Pull request merged**